.gitignore

```diff
@@ -39,4 +39,7 @@ yarn-error.log*
 
 # Emacs
 *~
-*#
\ No newline at end of file
+*#
+
+# firebase
+/firebase-export-1626142308993K36cpU/
```

Comment:
A couple things:
1. Where does this come from? Do you know which command created this folder?
2. It looks like this has a generated ID. This rule will not work if similarly created `/firebase-export-*/` folders are made.
3. There's already a Firebase section in this file. This belongs there if anything.